### DISCUSSION TITLE
How to define multiple actions in a route or page?

### DISCUSSION THREAD

**Discussion by tingfeng-key** [remix-run/remix#3914] [Q&A] [answered]
I now have a user list page (routing), and I want to add, delete, modify and check user data on this page. I checked the documentation and found no suitable guidance for this.
What should be done to achieve this requirement? Or is there any other better suggestion? thanks

**Comment by kiliman** [comment, marked as answer]
Ryan has a video describing a solution

_Remix Single: Multiple Forms and Single Button Mutations_
https://youtu.be/w2i-9cYxSdc

**Reply by tingfeng-key** [reply]
thank you, solved my problem
